src/oc_erchef/elvis.config

```diff
@@ -0,0 +1,30 @@
+[
+ {
+   elvis,
+   [
+    {config,
+     [
+      #{dirs => ["apps/*/src", "src"],
+        filter => "*.erl",
+        rules => [
+          {elvis_style, macro_names},
+          {elvis_style, no_if_expression},
+          {elvis_style, no_debug_call, #{ignore => [chef_test_db_helper,
+                                                    chef_test_suite_helper,
+                                                    oc_chef_wm_app]}},
+          {elvis_style, no_nested_try_catch},
+          {elvis_style, no_tabs},
+          {elvis_style, no_trailing_whitespace},
+          {elvis_style, operator_spaces},
+          {elvis_style, used_ignored_variable},
+          {elvis_style, variable_naming_convention}
```

Comment:
A personal preference is to add `use_your_spacebar`, but I imagine that would have quite a bit of required cleanup associated.